pages/collections/[id].tsx

```diff
@@ -214,7 +164,15 @@ const Home: NextPage<Props> = ({ fallback, id }) => {
               />
               <div className="col-span-full mx-6 mt-4 sm:col-end-[-1] md:col-start-4">
                 <div className="mb-4 hidden items-center justify-between md:flex">
-                  <div className="flex items-center gap-6">
+                  <div className="flex items-center gap-6 font-semibold">
+                    <RefreshButton
+                      refreshData={() => {
+                        tokens.setSize(1)
```

Comment:
I think we're better off just mutating all pages and refreshing all the data until we can get the cache busting to test if it works.